_posts/0000-01-02-smckenz6.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"Dont quit, suffer now and live the rest of your life as a champion" -Muhammed Ali